components/js/override.js

```diff
@@ -25,7 +25,10 @@ define(['angular'], function(angular) {
         'enablePushContentMenu': true,
       },
       'SERVICE_LOC': {
-        // 'messagesURL': 'staticFeeds/sample-messages.json',
+        'messagesURL': 'staticFeeds/sample-messages.json',
```

Comment:
`override.js` `SERVICE_LOC.messagesURL` changes in this changeset from [the change](https://github.com/uPortal-Project/uportal-app-framework/pull/611/files#diff-46d54ea0f3ac49e925d8968b0e250c2dR28) in #661 why?

Reply:
Switched it on during the tedium of resolving linting errors, forgot to switch it off.